core/storage/question/gae_models_test.py

```diff
@@ -511,3 +549,28 @@ def test_get_deletion_policy(self):
         self.assertEqual(
             question_models.QuestionRightsModel.get_deletion_policy(),
             base_models.DELETION_POLICY.LOCALLY_PSEUDONYMIZE)
+
+    def test_has_reference_to_user_id(self):
+        question_services.create_new_question_rights('question_id', 'owner_id')
+        self.assertTrue(
+            question_models.QuestionRightsModel
+            .has_reference_to_user_id('owner_id'))
+        # Change creator_id so we can test separate occurrences.
```

Comment:
I don't think I understand this comment. Could you explain in more detail?

Reply:
Done.